### ISSUE TITLE
Drag & Dropping Issue + Copying

### ISSUE THREAD

**Issue by Deanphillips** [livecomposer/Live-Composer#166]
Hey guys,

Having a few issues.

1) I can't drag and drop an element.

2) When I copy an element, I have to refresh the page for it to display correctly

See attached 
<img width="967" alt="screenshot 2016-08-17 11 55 59" src="https://cloud.githubusercontent.com/assets/21025321/17734074/f10b7ffc-6471-11e6-9ce6-fa2287dc6d90.png">
<img width="1440" alt="screenshot 2016-08-17 11 56 47" src="https://cloud.githubusercontent.com/assets/21025321/17734077/f2bf31d6-6471-11e6-9bb4-7144ded27763.png">

**Comment by jessicahawkins3344** [comment]
I am having trouble all together moving modules and elements, it's a nightmare.

Pretty much if I move one item on the page, nothing else will move until I refresh the page.

**Comment by crossvalley-studios** [comment]
It's a javascript error.

JavaScript error detected in a third-party plugin
Uncaught TypeError: a.addEventListener is not a function.

**Comment by jessicahawkins3344** [comment]
Do we have a fix?

**Comment by NGorco** [comment]
When you get error, after page refresh you can see red bug icon on the left side of settings panel. 
Click on it and paste here whole message text.

**Comment by jessicahawkins3344** [comment]
Getting this error while WP Editor is active: JavaScript error detected in a third-party plugin
Uncaught TypeError: Cannot read property 'value' of null
File "http://gaphc.smarkstaging.com/wp-content/plugins/wp-editor/extension

**Comment by jessicahawkins3344** [comment]
Still have error, however, when it's deactived too: it then changes to 👍 
`
JavaScript error detected in a third-party plugin
Script error.
File "", line 0, char 0`

**Comment by NGorco** [comment]
```
Uncaught TypeError: Cannot read property 'value' of null
File "http://gaphc.smarkstaging.com/wp-content/plugins/wp-editor/extension
```
path to that file telling, that error is causing wp-editor extension. 

Another problem with File "" much more interesting. We'll check it.

**Comment by rayd32** [comment]
I am also having the JS error as CROSS-VALLEY.  Is there a way for me to download the previous version?  I can't do anything with the new version and I just want to finish what I am doing to publish my site.  TIA

**Comment by jessicahawkins3344** [comment]
[live-composer-page-builder.zip](https://github.com/livecomposer/Live-Composer/files/422719/live-composer-page-builder.zip)

There is a version of 1.0.84

**Comment by rayd32** [comment]
Thanks Jess

**Comment by NGorco** [comment]
https://wordpress.org/plugins/live-composer-page-builder/developers/

here you can find all versions of LC.

**Comment by tbartels** [comment]
I've been having the same problems. I can sometimes drag a single item, but it's just the icon that gets dragged and left in the modules area.

```
Uncaught TypeError: a.addEventListener is not a function
File "http://voderetirement.wpengine.com/wp-content/plugins/live-composer-page-builder/js/builder.all.min.js?ver=1.1.1", line 3, char 699
```

 
<img width="964" alt="screen shot 2016-08-17 at 14 49 47" src="https://cloud.githubusercontent.com/assets/395000/17738842/df3ad5de-648a-11e6-94e4-1878038f8ff1.png">

**Comment by jessicahawkins3344** [comment]
Getting the exact same problem - seems like this version wasn't ready to go live

**Comment by jessicahawkins3344** [comment]
http://ddthemesdemo.com/volts/ is using the version before this new release, which worked pretty well.

**Comment by NGorco** [comment]
Update is coming soon. Fixed that problem. 

If you can't wait & have some skills in programming, try to set SCRIPT_DEBUG = true;

**Comment by akdevelopment** [comment]
Having same issue. Drag and drop functionality is completely gone. I disabled the only plugin that was putting out a java error and it did not help.

**Comment by akdevelopment** [comment]
Nevermind, this one just popped up
Live Composer returned JS error
TypeError: a.addEventListener is not a function
File "http://www.longbayaviation.dev/wp-content/plugins/live-composer-page-builder/js/builder.all.min.js?ver=1.1.1", line 3, char 697

**Comment by akdevelopment** [comment]
If I uninstall this version and install the old one again will I lose everything or will it be able to pull it back?

**Comment by livecomposer** [comment]
Bug fixed in version 1.1.2. Please update via your WP Admin.
Thank you for patience and your help.
